### DISCUSSION TITLE
How can i use raw sql

### DISCUSSION THREAD

**Discussion by sameddmr** [jasontaylordev/CleanArchitecture#792] [General]
I can't use raw sql with the context coming from IApplicationDbContext. Can anyone give me information on how to use it?

**Comment by apexdodge** [comment]
You can see an example here:

defined: https://github.com/RaythaHQ/raytha/blob/main/src/Raytha.Infrastructure/JsonQueryEngine/RaythaDbJsonQueryEngine.cs
used: https://github.com/RaythaHQ/raytha/blob/main/src/Raytha.Application/ContentItems/Queries/GetContentItems.cs

**Reply by sameddmr** [reply]
Thank you so much. I will test
